editor/plugins/org.fusesource.ide.projecttemplates/.descriptors/blank-spring.xml

```diff
@@ -0,0 +1,17 @@
+<assembly
+	xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3 http://maven.apache.org/xsd/assembly-1.1.3.xsd">
+	<id>blank-spring</id>
+	<includeBaseDirectory>false</includeBaseDirectory>
+	<formats>
+		<format>zip</format>
+	</formats>
+	<fileSets>
+		<fileSet>
+			<directory>${basedir}/../../../templates/blank/blank-spring</directory>
```

Comment:
We can do that but keep in mind that the template sources will be hidden in a 4th level subfolder instead of being a direct subfolder in the root of the repo. Thats probably not a problem for us but could confuse anyone else.